drafts/inlinemacros.md

```diff
@@ -165,99 +170,66 @@ positions:
 Inline types can specifically not be used as
 
  - types of abstract methods or fields
- - types of variables
- - types of class parameters
+ - types of vars
```

Comment:
I think, the word `variables` is too broad here. Some people in the Scala community call `vals` variables as well.